### ISSUE TITLE
Liquid 4.0 syntax errors

### ISSUE THREAD

**Issue by gvarisco** [italia/developers.italia.it#67]
I've started working on jekyll-3.5.x upgrade in a separate branch. When building the site with Liquid 4.0, we get a few warnings:

```
Liquid Warning: Liquid syntax error (line 3): Expected end_of_string but found id in "{{site.projects where: 'lang', active_lang | sort 'order' | reverse }}" in /_layouts/credits.html
Liquid Warning: Liquid syntax error (line 21): Expected end_of_string but found string in "{{site.projects | where: 'lang', active_lang | sort 'order' | reverse }}" in /_layouts/projects.html
Liquid Warning: Liquid syntax error (line 3): Expected end_of_string but found id in "{{site.projects where: 'lang', active_lang | sort 'order' | reverse }}" in /_layouts/credits.html
Liquid Warning: Liquid syntax error (line 21): Expected end_of_string but found string in "{{site.projects | where: 'lang', active_lang | sort 'order' | reverse }}" in /_layouts/projects.html
```

**Comment by gvarisco** [comment]
Commit 1864e99f456246ee3e4a9781a272438ecfe32888 and 416a0fc3de12a531a1c1175222a90f102fb16479 fix "Unexpected character ' " warnings

**Comment by gvarisco** [comment]
Fixed in 044fb9f12489ca8ac530ee585121e389e0cf50ea
